storage/src/system/auction/detail.rs

```diff
@@ -266,38 +266,58 @@ where
 }
 
 /// Returns the unbonding purses.
-pub fn get_unbonding_purses<P>(provider: &mut P) -> Result<UnbondingPurses, Error>
+pub fn get_unbonding_purses<P>(provider: &mut P) -> Result<BTreeMap<BidAddr, Unbond>, Error>
```

Comment:
If I am reading the function body correctly, we are getting the purses for the accounts only here, perhaps we should rename the function to match that, (nit)